### PR TITLE
added crunch to benchmarks

### PR DESCRIPTION
**Pull request by wanderer**

### CODE REVIEW

**Comment by indutny** [comment]
@wanderer could you please share the benchmark results here? Does it perform better than bn.js on some operations? It is placing any 1st places in benchmarks?

**Comment by wanderer** [comment]
Results: https://gist.github.com/wanderer/5aab77502625d6e2e1d5
> Does it perform better than bn.js on some operations?  

No
>It is placing any 1st places in benchmarks?  

No

Conclusion: crunch is not that fast.
Rational: crunch was featured in a weekly node.js email. I wanted to see how well it performed.

**Comment by Yaffle** [comment]
@wanderer , how to work with negative numbers? `crunch.stringify([-1])` and `crunch.parse("-1")` does not work for me
I have added `crunch` to the benchmark at http://yaffle.github.io/BigInteger/

**Comment by wanderer** [comment]
@Yaffle I don't know, I'm just trying to decided on a bignumber library myself.

File: benchmarks/index.js [hunk 1]
```diff
@@ -307,12 +337,17 @@ var prime4 = new bigi(
 var prime5 = new sjcl(
   'fffffffffffffffffffffffffffffffffffffffffffffffffffffffefffffc2f');
 
+var prime9 = crunch.parse('115792089237316195423570985008687907853269984665640564039457584007908834671663');
```
Comment (indutny) [inline benchmarks/index.js]: 80 column limit, please.

**Comment by indutny** [comment]
@wanderer I don't really want to add `crunch` at this point, what do you think about splitting the fix for a `.pow()` from this PR and putting it into a separate PR?

**Pull request closed without merging**